File: 2019/July(71)/lecture_001/T02_SI.java
import java.util.Scanner;

public class T02_SI{
    
   public static Scanner scn=new Scanner(System.in);

    public static void main(String[] args){
    System.out.println("please input value of p ,t and r");    
      int p=scn.nextInt();     
      int t=scn.nextInt();     
      int r=scn.nextInt();

      int si=(p*t*r)/100;
      System.out.println("SI is: " + si);    
    }
}

File: July(71)/lecture_001/T02_SI.cpp
#include<iostream>
using namespace std;


int main(int args,char** argv){
 
    int p,r,t;
    cout<<"please input value of p ,t and r"<<endl;
    cin>>p;
    cin>>t;
    cin>>r;

    int si=0;
    si=(p*t*r)/100;
    cout<<"SI is: "<<si;

    return 0;
}